sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedProcessorTest.java

```diff
@@ -64,7 +64,7 @@
 //    private final String hostName = "TestHost1";
     private final String hostName = RandomStringUtils.randomAlphabetic(6);
     private final int FEED_COUNT = 10;
-    private final int CHANGE_FEED_PROCESSOR_TIMEOUT = 5000;
+    private final int CHANGE_FEED_PROCESSOR_TIMEOUT = 5500;
```

Comment:
7000? Half of a second increase might not be enough...

Reply:
increased to 7000